lib/stack_master/stack_definition.rb

```diff
@@ -34,6 +35,7 @@ def initialize(attributes = {})
       @additional_parameter_lookup_dirs ||= []
       @template_dir ||= File.join(@base_dir, 'templates')
       @allowed_accounts = Array(@allowed_accounts)
+      @parameters ||= {}
```

Comment:
Why use the conditional assignment operator in the initializer? Is `@parameters` expected to be set before the initializer is run?

Reply:
The call to `super` above sets values from the yaml file, so it would either be set from that point or not, in which case the default is `{}`.